### DISCUSSION TITLE
p-dialog component can not be dragged and dropped in iPad

### DISCUSSION THREAD

**Discussion by hdthefirst** [primefaces/community#41] [PrimeNG] [answered]
Hello @cagataycivici,

I'm interested in learning the proper method for implementing the drag and drop functionality for the p-dialog component in an app using an iPad. 

Currently, dnd in iPad seems not worked for iPad so I have to disabled p-dialog dnd and added Angular CDK to the dialog header as a workaround, but I suppose there might be a better solution. 

Is there a possibility of an enhancement related to this in an upcoming update?

Thank you in advance.

Best regards,

Dong

**Comment by yigitfindikli** [comment]
Hi @Jeremy67,

This is by design. You can submit a feature request at https://github.com/primefaces/primeng/issues/new/choose.

**Comment by cagataycivici** [comment, marked as answer]
I think there is room for improvement using touch events for sure. An issue ticket would definitely help, we can tackle it after the accessibility enhancements. I think other Prime libs also have this limitation as core code is more or less the same.
